pkg/global/nucleusconstants.go

```diff
@@ -16,6 +16,7 @@ const (
 	RepoDir                  = HomeDir + "/repo"
 	CodeCoverageDir          = RepoDir + "/coverage"
 	DefaultHTTPTimeout       = 45 * time.Second
+	DefaultGitCloneTimeout   = 30 * time.Minute
```

Comment:
Caching workspace will also take longer than expected if the repo is too big in synapse